x-pack/plugins/integration_assistant/common/api/model/common_attributes.ts

```diff
@@ -21,6 +21,12 @@ export const PackageName = z.string().min(1);
 export type DataStreamName = z.infer<typeof DataStreamName>;
 export const DataStreamName = z.string().min(1);
 
+/**
+ * Blob containing the Base64 encoded rawSamples.
```

Comment:
What is the benefit of sending the base64-encoded string and not a regular string?

(Note also that the base-64 encoded string is 8÷6 larger than the regular one.)